lldb/source/Plugins/ExpressionParser/Swift/SwiftASTManipulator.cpp

```diff
@@ -197,7 +197,64 @@ do {
         weak_self ? "Swift.Optional where Wrapped == " : "";
 
     // The expression text is inserted into the body of $__lldb_wrapped_expr_%u.
-    wrapped_stream.Printf(R"(
+    if (options.GetEvaluateSelfAsGeneric()) {
+      // When evaluating as generic, we have to bypass the Swift typesystem.
+      // We do this by having $_lldb_expr call a sink function which takes
+      // $__lldb_arg and 2 type erased parameters (the pointer to self, and the
+      // pointer to the metadata). After generating LLVM IR we redirect the call
```

Comment:
Readability tip: Replace all instances of "we" with a noun. E.g., "SwiftASTManipulator", etc.